docs/source/examples/molecular_hydrogen.myst

```diff
@@ -85,31 +87,44 @@ radii = table[:, 0]
 coeffs = table[:, 1:]
 ```
 
-We define a function that evaluates the expectation value of the Hamiltonian on a noisy backend and for a given input circuit (ansatz).
-Here we use a Cirq density matrix simulator with a simple depolarizing noise model.
+We can also calculate their values automatically using OpenFermion (an open source quantum chemistry package).
+This results in slightly modified hamiltonians of the form:
+
+\begin{equation}
+    H = g_0 I + g_1 Z_0 + g_2 Z_1 + g_3 Z_0 Z_1 + g_4 X_0 X_1
```

Comment:
Do you have any intuition why the last term (g_5 Y_0 Y_1) is missing here?
If not, no worries. 
If yes, I would add a comment about it to make the reader more comfortable with the new different Hamiltonian.